ldm/generate.py

```diff
@@ -393,6 +392,7 @@ def process_image(image,seed):
         self.log_tokenization = log_tokenization
         self.step_callback = step_callback
         self.karras_max = karras_max
+        self.infill_method = infill_methods()[0], # The infill method to use
```

Comment:
This should use the `infill_method` parameter if it was passed in. Also, `self.infill_method` seems newly added here - is it correctly used later?

Reply:
This line was moved from the initialization into the call body so that patchmatch doesn't get called at load time. I've fixed up so that the line respects the passed value of `infill_method`.